.gitignore

```diff
@@ -2128,3 +2140,6 @@ packages/tools/website/utils/types.d.ts
 packages/tools/website/utils/types.js
 packages/tools/website/utils/types.js.map
 # AUTO-GENERATED - EXCLUDED TYPESCRIPT BUILD
+packages/app-mobile/components/get-responsive-value.test.js
+packages/app-mobile/components/get-responsive-value.test.js
+packages/app-mobile/components/get-responsive-value.test.js
```

Comment:
I don't think that should be here - run yarn run updateIgnored from the root

Reply:
Noted.

Reply:
I don't know why but running `updateIgnored` from root doesn't remove it from the .gitignore